static/js/components/EmailCompositionDialog.js

```diff
@@ -11,15 +11,15 @@ import type {
 const createDialogActions = (close, send) => ([
   <Button
     type="button"
-    className="dialog-button cancel-button"
+    className="mdl-button mdl-js-button cancel-button"
```

Comment:
We don't need to set the first two classes manually, these buttons will already have `mdl-button` and `mdl-js-button` on them (from the third party library)